content/docs/guides/pipelines/pipelines-metrics.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Pipeline metrics"
+description = "Export and visualize pipeline metrics."
+weight = 3
+toc = true
+
+[menu.docs]
+  parent = "pipelines"
+  weight = 6
++++
+This page is for component author to export metrics from the component. For details about how to build your own component, please see [Editing Build Your Own Component](/docs/guides/pipelines/build-component).
+ 
+## Overview of Metrics
+Kubeflow Pipeline supports scalar metrics exporting. Component author can write a list of metrics to describe the performance of the model to a local file which will later be uploaded as run-time metrics by the pipeline agent. The uploaded metrics will be visualized in the experiment runs table in the pipeline UI.
```

Comment:
To make the doc appeal more to the reader, let's address the reader directly rather than calling them "component author". So, change all "component author" to "you".

Reply:
done